### PR TITLE
Mojmap class names

### PR DESCRIPTION
**Pull request by fxmorin**
This is based on Altrisi's mojmap branch https://github.com/altrisi/fabric-carpet/tree/mojmap
It adds the class names in a separate PR for PR https://github.com/gnembon/fabric-carpet/pull/1288

### CODE REVIEW

File: src/main/java/carpet/mixins/CoralFanBlock_renewableCoralMixin.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import net.minecraft.world.level.levelgen.feature.CoralMushroomFeature;
+import net.minecraft.world.level.levelgen.feature.CoralTreeFeature;
+import net.minecraft.world.level.levelgen.feature.configurations.NoneFeatureConfiguration;
+import net.minecraft.world.level.material.MaterialColor;
+import org.spongepowered.asm.mixin.Mixin;
+
+import java.util.Random;
+
+@Mixin(CoralFanBlock.class)
+public abstract class CoralFanBlock_renewableCoralMixin implements BonemealableBlock
```
Comment (altrisi) [inline src/main/java/carpet/mixins/CoralFanBlock_renewableCoralMixin.java:28]: Looks like this class was completely recreated...
Line endings, I'd guess? Or something
(https://github.com/altrisi/fabric-carpet/compare/mojmap...fxmorin:mojmap-class-names#diff-280cecdaf4cb09cc4e0e4ecc0114774572ab5662ec3868ff725b716cf401fbf7)

**Comment by altrisi** [comment]
Aaaand as expected this now has conflicts (in the form of missing commits)

**Comment by fxmorin** [comment]
Don't worry about it. I will rebase everything once you are done making changes. This is basically a placeholder at the moment

**Comment by gnembon** [comment]
please rebase now then

**Comment by fxmorin** [comment]
Was unable to rebase, so instead, it was cherry-picked onto another branch. #1301

**Pull request closed without merging**